spec/rack/twilio_webhook_authentication_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 require 'rack/mock'
+require 'rack/twilio_webhook_authentication'
```

Comment:
Why is this line added? Is this interfering with any of the added tests?

Reply:
yes